dot/network/host.go

```diff
@@ -76,8 +76,16 @@ func newHost(ctx context.Context, cfg *Config) (*host, error) {
 		return nil, err
 	}
 
+	dhtOpts := []kaddht.Option{
+		kaddht.Datastore(dsync.MutexWrap(ds.NewMapDatastore())),
+		kaddht.Mode(kaddht.ModeAutoServer),
```

Comment:
curious why this is changed?

Reply:
Default DHT mode is `ModeAuto` which starts it as a client and it cannot respond to queries. https://github.com/libp2p/go-libp2p-kad-dht/blob/09d923fcf68218181b5cd329bf5199e767bd33c3/dht_options.go#L28